.circleci/config.yml

```diff
@@ -133,12 +133,36 @@ jobs:
           name: Login for Docker
           command: echo $DOCKER_TOKEN | docker login -u "$DOCKER_USER" --password-stdin
       - run:
-          name: API Governance Compliance
+          name: Create version.txt
           command: |
-            docker pull qlik/asmquery
+              set -x
+              if [ -n "${CIRCLE_TAG}" ]; then
+                  version=${CIRCLE_TAG#v}
+              else
+                  version=$(git describe --tags --abbrev=7 --match "v*")
+                  version=${version#v}
+              fi
+              echo "$version" > ./version.txt
+              echo "Building $version"
+      - run:
+          name: Prepare API Compliance
+          command: |
+            docker pull ghcr.io/qlik-trial/api-compliance
             docker create -v /specs --name specs alpine:3.4 /bin/true
             docker cp ./apis/stardust/api-spec/spec.json specs:/specs
-            docker run --volumes-from specs -e target=Colony -e runMode=failBuild -e apiId=abdc47c2-921a-45d3-ba5c-80d448574c5b -e specPath='/specs/spec.json' -e commit="$CIRCLE_SHA1" qlik/asmquery
+      - run:
+          name: Run API Compliance
+          command: >
+            VER=$(cat ./version.txt)
+
+            docker run --volumes-from specs
+            -e SPEC_PATHS="abdc47c2-921a-45d3-ba5c-80d448574c5b@/specs/spec.json"
+            -e COMMIT_SHA="$CIRCLE_SHA1"
+            -e RELEASE_TAG="$VER"
+            -e CREDENTIALS_S3_SECRETKEY="$APICULTURIST_S3"
+            -e CREDENTIALS_GITHUB="$APICULTURIST_GITHUB"
+            -e CREDENTIALS_COLONY="$APICULTURIST_TOKEN"
```

Comment:
The APICULTURIST_* secrets needs to be added to qlik-oss organisation or nebula.js in circelci

Reply:
We added an api-compliance context to OSS and I added it to the api governance job. Let me know if this fixes the issue.

Reply:
(I will not know if it actually works until after the PR is merged, as it is not run on PRs from forks)
but the context is named `api-compliance` i circelci but the this config use the name `api-governance` (I expect that to be a problem)

Reply:
100% my fault! Thank you for pointing it out. I'm going to fix it. If you want, you can give me access to the repository and I can recreate the branch there.